document shared memory segment issue (#6546)
digital-asset/daml: README.md

by overwriting these configs in either a `.bazelrc.local` file in each copy, or a `~/.bazelrc` file
in your home directory.

### Shared memory segment issues

> On macOS at least, it looks like our setup is not always properly closing all
> the resources PostgreSQL uses. After a number of test runs, you may encounter
> an error message along the lines of:

Reviewer: ```suggestion
On macOS at least, it looks like our setup does not always properly close the
resources PostgreSQL uses. After a number of test runs, you may encounter an
error message along the lines of:
```
